datalad_gooey/dataladcmd_exec.py

```diff
@@ -21,6 +21,7 @@ class GooeyDataladCmdExec(QObject):
     # thread_id, cmdname, cmdargs/kwargs
     execution_started = Signal(str, str, dict)
     execution_finished = Signal(str, str, dict)
+    execution_failed = Signal(str, str, dict, str)
```

Comment:
Would be good to add a comment on the semantics of the parameters. There is an additional `str`